spec/display_spec.rb

```diff
@@ -28,20 +28,30 @@
       end.to output('Choose a position from 1-9: ').to_stdout
     end
 
-    it 'returns the users move as an integer' do
+    it 'returns the users move' do
       allow($stdin).to receive(:gets).and_return('1')
       move = display.ask_for_move
-      expect(move).to eq(1)
+      expect(move).to eq('1')
     end
   end
 
   context 'Displaying the current player:' do
     display = Display.new
```

Comment:
I think you can have one Display per the whole controller_spec file, no need to create a new one for each context.

Reply:
Hmm - I think you're right. But I prefer to minimise global state in tests. Think it's a style thing mostly.